docs/qingstor_service_usage.md

```diff
@@ -186,3 +186,42 @@ output = bucket.abort_multipart_upload(
 # Example: 400
 print(output.status_code)
 ```
+
+GET Object Download Url Example
+
+1. Initialize the Bucket service with accesskeyid and secretaccesskey.
+
+    ```python
+    import time
+
+    from urllib.request import urlretrieve
+    from qingstor.sdk.service.qingstor import QingStor
+    from qingstor.sdk.config import Config
+    from urllib.request import pathname2url
+
+    config = Config('MKWSFQWXTOKCQKDEGBZP', 'qAHKIn11AQAHRyIPNEQViG7rHmCUEMvyiY0YoqiV')
+    qingstor = QingStor(config)
+    bucket_api = qingstor.Bucket("zzz-taiyou-03", "pek3b")
```

Comment:
**Do not use real access key and bucket please**

Update this file and reset your access key as soon as possible.